src/libraries/Microsoft.Extensions.Options/gen/Emitter.cs

```diff
@@ -484,32 +484,38 @@ public void EmitRangeAttribute(string modifier, string prefix, string className,
         public override string FormatErrorMessage(string name) =>
                 string.Format(global::System.Globalization.CultureInfo.CurrentCulture, GetValidationErrorMessage(), name, Minimum, Maximum);
         private bool NeedToConvertMinMax { get; }
```

Comment:
For consistency, can we make this `private readonly bool _needToConvertMinMax;`? (Same in the other files)

Reply:
Done.